.gitignore

```diff
@@ -97,4 +97,6 @@ ENV/
 # Generated files by framework
 examples/baseruntime/generated.py
 examples/memcached/generated.py
+downloaded_git
```

Comment:
you can use `.git/info/exclude` to ignore your local files, no need to store those in repo-wide .gitignore